met-web/src/components/engagement/banner/BannerWithoutImage.tsx

```diff
@@ -63,24 +62,13 @@ const BannerWithoutImage = ({ savedEngagement }: BannerProps) => {
                         rowSpacing={2}
                     >
                         <Grid item xs={12} sx={{ maxHeight: '20em', overflowY: 'auto', overflowX: 'auto' }}>
-                            {isSmallScreen ? (
-                                <MetHeader2 sx={{ mb: 1 }}>{name}</MetHeader2>
-                            ) : (
-                                <MetHeader1 sx={{ mb: 1 }}>{name}</MetHeader1>
-                            )}
-
+                            <MetHeader1>{name}</MetHeader1>
                             <Editor editorState={getEditorState(rich_description)} readOnly={true} toolbarHidden />
                         </Grid>
                         <Grid item xs={12}>
-                            {isSmallScreen ? (
-                                <MetHeader3 style={{ fontWeight: 600 }} color="black">
-                                    {EngagementDate}
-                                </MetHeader3>
-                            ) : (
-                                <MetHeader2 style={{ fontWeight: 600 }} color="black">
-                                    {EngagementDate}
-                                </MetHeader2>
-                            )}
+                            <Typography sx={{ fontWeight: 'bold' }} variant="subtitle1" color="black">
```

Comment:
have a feeling this is MetBody. If not can we please add a subtitle component so we can reuse it

Reply:
good idea